### DISCUSSION TITLE
How restrict target branch for PRs?

### DISCUSSION THREAD

**Discussion by buhtz** [community/community#44003] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hello,

I would like that contributors who are not admins of the repo are only able to open PR's against the branch named `develop`. This should be the default setting when opening a PR and shouldn't be customizable by the PR openers.

How can I do that?

I assume this is hidden somewhere in _Branch protection rules_, right?
But I'm a bit overwhelmed by the possibilities there and not even sure if I can reach my goal with that part of the settings.

**Comment by buhtz** [comment]
Does no one has an idea about it?

**Comment by buhtz** [comment]
Hello!? Knock knock!

**Comment by patrick-knight** [comment, marked as answer]
For any branch you want to require a PR on you'd target that branch or any branches that you'd like to protect.

For example if you were to target `**/*` you could protect every branch recursively and then define the setting in `Require a pull request before merging` that are relevant for you. 

If you only wanted to protect `develop` and no other branch create one branch protection rule for that branch and define the relevant PR rules. 

To have multiple rules on one branch they are exclusive of each other, so if you have rules for `*` and `develop` they will not overlap. 

Hope this helps! And thanks for being patient we get tons of questions here and sometimes they slip by.
